pokemongo_bot/cell_workers/soft_ban_worker.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from pokemongo_bot.cell_workers.utils import distance
+from pokemongo_bot.worker_result import WorkerResult
+from pokemongo_bot.cell_workers import SeenFortWorker
+from pokemongo_bot.cell_workers import MoveToFortWorker
+
+
+class SoftBanWorker(object):
+    def __init__(self, bot):
+        self.running = False
+        self.bot = bot
+    def work(self):
+       if self.bot.softbanned == True:
+           nearest_fort = self.nearest_pokestop()
+           MoveToFortWorker(nearest_fort, self.bot).work()
```

Comment:
You don't need this. The only way to get into the softbanned mode is to have spun a fort and not gotten anything. That means you are close enough to spin it again.